### PR TITLE
added users translations to ingredients taxonomy

### PR DESCRIPTION
**Pull request by stephanegigandet**

### CODE REVIEW

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -240,6 +239,7 @@ stopwords:zh::配料
 
 
 fr:additifs alimentaires
+	'dont acides gras satur(e|é)s
```
Comment (VaiTon) [inline taxonomies/ingredients.txt]: Why is this here?

Comment (stephanegigandet) [reply taxonomies/ingredients.txt]: I don't know, some bug with my script to add the translations. I fixed it manually.
